runner.rb

```diff
@@ -1,9 +1,9 @@
 require './lib/stat_tracker'
 require 'CSV'
 
-game_path = './data/games_test.csv'
-team_path = './data/teams_test.csv'
-game_teams_path = './data/game_teams_test.csv'
+game_path = './data/games.csv'
```

Comment:
Be sure that we change these back to the test(fixture) file paths before continuing the project